addons/web/static/tests/helpers/mock_server.js

```diff
@@ -2187,6 +2212,13 @@ export class MockServer {
     }
 }
 
+// mockRegistry allows to register mock version of methods or routes,
+// for all models:
+//   MockServer.mockRegistry.add('some_route', () => "abcd");
+// for a specific model (e.g. 'res.partner'):
+//   MockServer.mockRegistry.add('res.partner/some_method', () => 23);
+MockServer.mockRegistry = new Registry();
```

Comment:
I'd use a category instead (so nothing required here, simply use above `registry.category("mock_server")`. We should do the same for the sample server.